x/marker/keeper/keeper_test.go

```diff
@@ -128,6 +128,88 @@ func TestAccountKeeperReader(t *testing.T) {
 	require.EqualValues(t, count, 1)
 }
 
+func TestKeeperUsdfMigrate(t *testing.T) {
+	//app, ctx := createTestApp(true)
+	app := simapp.Setup(false)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+
+	addr := types.MustGetMarkerAddress("usdf.c")
+	user := testUserAddress("test")
+	// create account and check default values
+	mac := types.NewEmptyMarkerAccount(
+		"usdf.c",
+		user.String(),
+		[]types.AccessGrant{*types.NewAccessGrant(user, []types.Access{types.Access_Mint})})
+
+	require.NoError(t, app.MarkerKeeper.AddMarkerAccount(ctx, mac))
+
+	m, err := app.MarkerKeeper.GetMarkerByDenom(ctx, "usdf.c")
+	require.NoError(t, err)
+	require.NotNil(t, m)
+	require.EqualValues(t, m.GetDenom(), "usdf.c")
+	require.EqualValues(t, m.GetAddress(), addr)
+
+	m, err = app.MarkerKeeper.GetMarker(ctx, addr)
+	require.NoError(t, err)
+	require.NotNil(t, m)
+	require.EqualValues(t, m.GetDenom(), "usdf.c")
+	require.EqualValues(t, m.GetAddress(), addr)
+	require.EqualValues(t, m.GetMarkerType(), types.MarkerType_Coin)
+
+	count := 0
+	app.MarkerKeeper.IterateMarkers(ctx, func(record types.MarkerAccountI) bool {
```

Comment:
we don't really need this check as denom is enforced to be the address of the marker and thus must be unique on chain as address is the primary key/identifier for accounts.

Reply:
this was just an additional check that the earlier tests were doing ...i can remove it 🤷